feature/visited/src/main/kotlin/uk/govuk/app/visited/ui/EditVisitedScreen.kt

```diff
@@ -293,21 +321,39 @@ private fun BottomNavBar(
             verticalAlignment = Alignment.CenterVertically,
             horizontalArrangement = Arrangement.SpaceBetween
         ) {
-            TextButton(
-                onClick = { /* TODO */ }
-            ) {
-                BodyRegularLabel(
-                    text = selectText,
-                    color = GovUkTheme.colourScheme.textAndIcons.link,
-                    textAlign = TextAlign.Start
-                )
+            if (uiState?.hasAllSelectedItems == true) {
```

Comment:
We could reduce lines and duplication here here by doing something like:

val onClick = if (uiState?.hasAllSelectedItems == true) onDeselectAll else onSelectAll
val text = if (uiState?.hasAllSelectedItems == true) deselectText else selectText

Reply:
And perhaps first create a val for `uiState?.hasAllSelectedItems == true` also might be cleaner for above example...

Reply:
I think this is the kind of thing your looking for

Reply:
Fixed up